### PR TITLE
更正了readme中的小问题

### PR DESCRIPTION
**Pull request by BubbleSci**
修改了readme中的问题：
代码缩进和标签问题
packages中compiler-postcss中readme的格式问题

##### Checklist

- [ ] `npm run test` passes
- [ ] tests and/or benchmarks are included
- [ ] cases or donate is changed or added
- [ ] documentation is changed or added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,22 +35,24 @@ WePY (发音: /'wepi/)是一款让小程序支持组件化开发的框架，通
 <style lang="less">
   @color: #4D926F;
   .num {
-      color: @color;
+  color: @color;
```
Comment (Gcaufy) [inline README.md:38]: indent

File: README.md [hunk 2]
```diff
@@ -33,22 +35,24 @@ WePY (发音: /'wepi/)是一款让小程序支持组件化开发的框架，通
...
   }
 </style>
 <template>
   <div class="container">
     <div class="num" @tap="num++">
       {{num}}
     </div>
+    <customComponent></customComponent>
```
Comment (Gcaufy) [inline README.md:46]: custom-component

File: README_EN.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    },
+  });
+</script>
+```
+
+### Install and use
+
+#### Install (update) the WEPY command-line tool.
+
+```console
```
Comment (Gcaufy) [inline README_EN.md:77]: bash

File: README_EN.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  wepy.page({
+    data: {
+      num: 0,
+      text: 'Hello World',
+    },
+  });
+</script>
+```
+
+### Install and use
```
Comment (Gcaufy) [inline README_EN.md:73]: Usage

File: README_EN.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+#### Install (update) the WEPY command-line tool.
```
Comment (Gcaufy) [inline README_EN.md:75]: Install (Upgrade)

File: README_EN.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+```console
+npm install @wepy/cli@next -g
+```
+
+#### Generate development instances
```
Comment (Gcaufy) [inline README_EN.md:81]: Get start with an example

File: README_EN.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+#### Install dependencies
+
+```console
+cd myproject
+npm install
+```
+
+#### Develop real-time compilation
```
Comment (Gcaufy) [inline README_EN.md:94]: Watch mode

File: README_EN.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+```console
+wepy build --watch
+```
+
+#### Developer tools import project
```
Comment (Gcaufy) [inline README_EN.md:100]: Import project

File: README_EN.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+爽到家小程序、
+[应用全球排行](https://github.com/szpnygo/wepy_ios_top)[（开源）](https://github.com/szpnygo/wepy_ios_top)、
+[we 川大](https://github.com/mohuishou/scuplus-wechat)[（开源）](https://github.com/mohuishou/scuplus-wechat)、
+聊会儿、
+[诗词墨客](https://github.com/huangjianke/weapp-poem)[（开源）](https://github.com/huangjianke/weapp-poem)、
+[南京邮电大学](https://github.com/GreenPomelo/Undergraduate)[（开源）](https://github.com/GreenPomelo/Undergraduate)
+
+...
+
+### communication group
```
Comment (Gcaufy) [inline README_EN.md:155]: Wechat Group

File: README_EN.md [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+
+WePY group has reached 500 members，Please add gcaufy_helper friends or scan the code to add friends, verify the reply 'wepy' according to the reference to enter the group.
+
+![wepy_qr_code](https://user-images.githubusercontent.com/2182004/82732473-feb50c80-9d3f-11ea-9a5f-0efc6ce40f74.png)
+
+### Participate in the contribution
```
Comment (Gcaufy) [inline README_EN.md:161]: Contribution

File: README_EN.md [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+
+If you have any comments or Suggestions, please feel free to contribute to improving the WeChat app-development experience by asking for Issues or Pull Requests.<br>see details：[CONTRIBUTING.md](./CONTRIBUTING.md)
+
+[Tencent Open Source Incentive Plan](https://opensource.tencent.com/contribution) EncouraTencent Open Source Incentive Plange developers to participate and contribute. Look forward to your participation.
+
+### Links
+
+[Documentation](https://tencent.github.io/wepy/)
```
Comment (Gcaufy) [inline README_EN.md:169]: link 不对。

File: packages/use-promisify/README.md [hunk 12]
```diff
@@ -64,9 +64,9 @@ func({
 wepy.promisify(func)({key: 'mykey'}).then(console.log).catch(console.error);
 ```
 
-#### error-first
+#### 错误优先
```
Comment (Gcaufy) [inline packages/use-promisify/README.md:67]: error-first

File: README.md [hunk 13]
```diff
@@ -31,24 +33,26 @@ WePY (发音: /'wepi/)是一款让小程序支持组件化开发的框架，通
...
-      color: @color;
+  color: @color;
   }
 </style>
 <template>
   <div class="container">
     <div class="num" @tap="num++">
       {{num}}
     </div>
+    <custom-Component></custom-Component>
```
Comment (Gcaufy) [inline README.md:46]: 小写的c

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2661) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2661) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/wepy?pullRequest=2661) it.</sub>

**Comment by BubbleSci** [comment]
好的已经修改了



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Gcaufy"<notifications@github.com&gt;; 
发送时间: 2020年7月26日(星期天) 下午4:05
收件人: "Tencent/wepy"<wepy@noreply.github.com&gt;; 
抄送: "带鱼希儿呀"<1228160604@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [Tencent/wepy] 更正了readme中的小问题 (#2661)





 
@Gcaufy requested changes on this pull request.
 
 
In README.md:
 &gt;    }  </style&gt;  <template&gt;    <div class="container"&gt;      <div class="num" @tap="num++"&gt;        {{num}}      </div&gt; +    <custom-Component&gt;</custom-Component&gt;  
小写的c

**Pull request merged**